ci/jobs/fast_test.py

```diff
@@ -231,6 +230,11 @@ def main():
         results.append(
             Result.create_from(name=step_name, status=res, stopwatch=stop_watch_)
         )
+        if not results[-1].is_ok():
+            attach_files.append(f"{temp_dir}/build/programs/clickhouse")
+            attach_files.append(
+                f"{temp_dir}/var/log/clickhouse-server/clickhouse-server.err.log"
```

Comment:
Odd, why we don't add the complete log? It was there before for fast-test as well. And it is impossible to debug without it